packs/data/equipment-effects.db/effect-owlbear-claw.json

```diff
@@ -0,0 +1,52 @@
+{
+    "_id": "LHrEiMzmjU6NRQSw",
+    "img": "systems/pf2e/icons/equipment/consumables/talismans/owlbear-claw.webp",
+    "name": "Effect: Owlbear Claw",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Owlbear Claw]{Owlbear Claw}</p>\n<p>The triggering attack gains the weapon's critical specialization effect.</p>"
+        },
+        "duration": {
+            "expiry": null,
+            "sustained": false,
+            "unit": "unlimited",
```

Comment:
Probably want a duration.

Reply:
In this case, we don't have a duration that works for this. Once affixed, when you crit with the weapon you would drag this effect on the actor which is the act of envisioning, and then roll your crit damage for the crit spec to be triggered. the owlbear claw is then immediately consumed so I left this as unlimited to be manually removed.